rero_ils/modules/patrons/jsonschemas/patrons/patron-v0.0.1.json

```diff
@@ -275,6 +276,60 @@
           "templateOptions.required": "model.roles.some(v => v === 'patron')"
         }
       }
+    },
+    "blocking": {
```

Comment:
Do you need to keep a history of the block codes? because you can make it simpler like this:
```
"blocking": [{"code": "regular", "note": "reason to block", "active": True}]
```
this will allow you to have disable/enable each block and have a note for each block.
and if one or more blocks are active then patron is blocked.

Reply:
Not history, but a list of multiple codes. Is that not possible as I do it?